src/module-elasticsuite-analytics/view/adminhtml/templates/search/usage/terms/no_results.phtml

```diff
@@ -26,6 +26,9 @@ $hiddenTermsData    = array_slice($termsData, $maxVisibleTerms);
 <div class="dashboard-item" id="<?= $block->getJsId() ?>">
     <div class="dashboard-item-title">
         <span><?= $block->getTitle() ;?></span>
+        <span id="<?= $block->getJsId() ?>_copy" class="admin__field-action-export" title="<?= $block->escapeHtml(__('Copy table to clipboard')) ?>">
+            <span class="copy-feedback-text"><?= __('Copied to clipboard !') ?></span>
+        </span>
```

Comment:
Copy feature to remove from here, it does not make sense to offer it on an empty table